docs/compendium/substitute.md

```diff
@@ -25,7 +36,90 @@ introduction...
 ### Error responses
 
 ```json
+401 Unauthorized
+
+{"error":"not authenticated or not allowed"}
```

Comment:
Use two different error messages for this, please. Same HTTP code, but distinguish the meanings.